From 75909b8952ab6def75532c4555a33ec9a46e3422 Mon Sep 17 00:00:00 2001
From: Peleg Vigodny <peleg.vigodny@tandemg.com>
Date: Thu, 07 May 2020 10:40:05 +0300
Subject: [PATCH] [SW-10967] gaudi: linux: qsfp-dd monitor: enable

Enable HL QSFP-DD hardware monitor in Gaudi Linux.

This driver will add temperature sensors for the two QSFP-DD modules
connected to HL200, and will allow to power modules on/off and display
present modules and their vendors.

Change-Id: I487d3775be6d1fd17e348645adf78c196a12eac5
Signed-off-by: Peleg Vigodny <peleg.vigodny@tandemg.com>
---

diff --git a/configs/gaudi_defconfig b/configs/gaudi_defconfig
index 759a1a8..f3953cb 100644
--- a/configs/gaudi_defconfig
+++ b/configs/gaudi_defconfig
@@ -63,3 +63,4 @@
 CONFIG_GPIOLIB=y
 CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_PCA953X=y
+CONFIG_SENSORS_QSFP_DD_MON=y
